src/lib/get-verified-fetch.ts

```diff
@@ -68,8 +68,11 @@ export async function getVerifiedFetch (config: ConfigDb, logger: ComponentLogge
   } else {
     config.routers.forEach((router) => {
       routers.push(delegatedHTTPRouting(router, {
+        // NOTE: in practice 'transport-ipfs-gateway-http' exists only in IPNI results, we won't have any DHT results like this unless..
+        // TODO: someguy starts doing active probing to identify peers which have funcitonal HTTP gateway
```

Comment:
we should probably link to a someguy issue

Reply:
```suggestion
        // TODO: someguy starts doing active probing (https://github.com/ipfs/someguy/issues/53) to identify peers which have functional HTTP gateway 
```